dimagi/pages/static/style/errors.css

```diff
@@ -0,0 +1,92 @@
+.error-404-page .hero {
```

Comment:
why are all these css files being committed? We use `.scss`. Instructions for how to setup the repo locally are provided in the `README` at the root of the repository.